rbc/tests/heavydb/test_array_methods.py

```diff
@@ -1,7 +1,8 @@
 import pytest
 import numpy as np
 from rbc.tests import heavydb_fixture
-from rbc.heavyai import Array
+from rbc.heavydb import Array
+from rbc.stdlib import array_api  # noqa: F401
```

Comment:
Why is this needed?

Reply:
This initializes the array methods implementations, otherwise the tests will fail.